cli/errors.go

```diff
@@ -20,6 +20,7 @@ const (
 	errInvalidLensConfig        string = "invalid lens configuration"
 	errSchemaVersionNotOfSchema string = "the given schema version is from a different schema"
 	errRequiredFlag             string = "the required flag [--%s|-%s] is %s"
+	errInvalidAscensionOrder    string = "invalid order for field %s: expected ASC or DESC"
```

Comment:
I have now made this change.

Thanks for the heads up -- you are right, that I did it that way originally because of how I saw errRequiredFlag done. But I see this is cleaner indeed.